lamindb_setup/_init_instance.py

```diff
@@ -339,6 +340,7 @@ def init(
         if _write_settings:
             settings.auto_connect = True
         importlib.reload(importlib.import_module("lamindb"))
+        reset_django_module_variables()
```

Comment:
i don't think it should always reset, it should do the reset only on re-connect. Also not clear why `importlib.reload` deosn't do this already.